### PR TITLE
Squash all and remove binary file

### PR DESCRIPTION
**Pull request by Erikvv**
* squash all PR's into one commit (discussed with @kaloyan-raev beforehand)
* remove uplink-c binary from git
    * build zip file in Jenkins
    * change installation instructions in README.md
    * change update instructions in MAINTAINERS.md
* Version bumps of build dependencies (go, php) and small textual changes.

### CODE REVIEW

**Comment by stefanbenten** [comment]
@Erikvv Heads up, i just alligned the repo to use `main` as default branch and removed the offending `master` as default.
I think/hope, that i adjusted everything accordingly.

**Comment by D4rk4** [comment]
Hurry up!

**Comment by Erikvv** [comment]
@D4rk4 I don't think there will be any major changes. If you are waiting for this if I would just start development. 

The install instructions at https://github.com/storj-thirdparty/uplink-php/blob/eeae9cefae09831cb1cefe42159d33d82c7bffe5/README.md#installation work. The instructions I gave on the forum also still work until the branches are cleaned up.

Please file an issue if you run into any trouble I'll look into it ASAP.

Is there any documentation you find missing?

What software will you be making if I may ask?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+```
+
+3. Install using composer. Copy and run:
+
+```
+composer config repositories.storj/uplink '{
+    "type": "package",
+    "package": {
+        "name": "storj/uplink",
+        "version": "1.0.0",
```
Comment (Erikvv) [inline README.md:36]: What version do we release initially? 1.0.0 or 0.1.0?

Comment (egonelbre) [reply README.md:36]: I would recommend 0.1.0 (or alternatively 1.0.0-rc) for a month... and let people test it out to find things that were missed and that may require breaking api changes.

Comment (Erikvv) [reply README.md]: Changed to 0.1.0

**Review by kaloyan-raev — APPROVED** [review]
@Erikvv Thanks a lot for preparing this PR! All looks good to me.

Perhaps, the only thing I miss in README is to clarify the OS platform that is currently supported. I guess this is Linux and64, right? (limited by the included build variant of libuplink.so)

**Comment by Erikvv** [comment]
> Perhaps, the only thing I miss in README is to clarify the OS platform that is currently supported. I guess this is Linux and64, right? (limited by the included build variant of libuplink.so)

Added Linux x64 to the requirements section. 

I have an ARM v8 board here I'll give a try at cross-compiling.

**Review by kaloyan-raev — APPROVED** [review]

**Review by egonelbre — APPROVED** [review]

**Comment by Erikvv** [comment]
Jenkins being a bit random.

**Comment by Erikvv** [comment]
Changed 
```
agent {
    docker {
        image docker.build('storj-php').id
        args '-u root:root'
    }
}
```
to 
```
agent {
    dockerfile {
        args '-u root:root'
    }
}
```

**Review by kaloyan-raev — APPROVED** [review]

**Review by egonelbre — APPROVED** [review]

**Pull request merged**